tests/README.md

```diff
@@ -293,7 +293,7 @@ To quickly open the tests for a rule with ID, add a similar function to your `.b
 function open_tests() {
     rule_id="$1"
     ssg_root="/path/to/your/content/git/repository"
-    tests_dir=$(find $ssg_root/tests/data/ -name rule_$rule_id)
+    tests_dir=$(find $ssg_root/tests/data/ -name *$rule_id*)
```

Comment:
This change is incompatible with subsequent command `cd $tests_dir`. 
If there are multiple matches:
```shell
bash: cd: too many arguments
```

Reply:
If `find` actually returns a file, instead of a directory.
```bash
bash: cd: data/group_system/group_auditing/group_auditd_configure_rules/group_audit_file_modification/rule_audit_rules_unsuccessful_file_modification_unlink/one_sys_with_filter.fail.sh: Not a directory
```